src/wfprint.c

```diff
@@ -20,7 +20,7 @@
 DWORD
 PrintFile(HWND hwnd, LPTSTR szFile)
 {
-   DWORD          ret;
+   DWORD_PTR     ret;
    INT           iCurCount;

```

Comment:
DWORD (original) seems better here.  Any reason why DWORD_PTR?:

Reply:
ShellExecuteEx returns an HINSTANCE, which is a pointer type.  It's not a real HINSTANCE, but the compiler doesn't know that.  Prior to this change the compiler thought this was pointer truncation - casting a 64 bit pointer to a 32 bit integer.  We could suppress the warning with a double cast (cast to DWORD_PTR then DWORD), but it's not clear to me that this would be correct.  ShellExecuteEx is supposed to return a value above 32 to indicate success, but is a value above 2^32 supposed to indicate success?